app/routes/screens/performanceMeter/colors.js

```diff
@@ -0,0 +1,14 @@
+export default {
```

Comment:
great idea. 
this is file is about to be removed. It was just a test for testing d3 and ART